pkg/pdfcpu/writeObjects.go

```diff
@@ -673,6 +673,9 @@ func writeObjectGeneric(ctx *model.Context, o types.Object, objNr, genNr int) (e
 	case types.Name:
 		err = writeNameObject(ctx, objNr, genNr, o)
 
+	case *types.LazyObjectStreamObject:
```

Comment:
This kind of stands out here because of the pointer usage.
Do you think it makes sense to pass the object instead for consistency? Trade offs?

Reply:
Changed